common/src/model_registration/register.py

```diff
@@ -152,7 +150,10 @@ def main(args):
     # Updating tags and properties with value provided in metadata file
     if args.model_metadata:
         with open(args.model_metadata, "r") as stream:
-            metadata = json.load(stream)
+            try:
```

Comment:
You can use `yaml.safe_load` to load both JSON and YAML data

Reply:
Thanks ayush . Updated